modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/js/style-book-editor/PreviewInfoBar.js

```diff
@@ -14,7 +14,7 @@
 
 import ClayIcon from '@clayui/icon';
 import ClayPopover from '@clayui/popover';
-import {Align} from 'metal-position';
+import {Align, align} from 'frontend-js-web';
```

Comment:
Isn't this a bit confusing as far as APIs go? What's the difference between `Align` and `align`?

Reply:
Nice catch, we can probably call `Align` `ALIGN_POSITIONS` (it's an object of constants), while `align` is the method to align.

Reply:
Yes I agree that this is confusing as far as APIs go.

`Align` used to be class with constants and (only) static methods - with this change `Align` is just an object that exposes constants and the static methods are now just functions that we export (such as `align`)

I agree with what @carloslancha suggested, `ALIGN_POSITIONS` sounds like a better name.